changelog.d/1519.change.rst

```diff
@@ -0,0 +1 @@
+Perform additional path normalization to ensure path values to a directory is always the same, preventing false positives when checking scripts have a consistent prefix to set up on Windows.
```

Comment:
Oooh, a chance to use the new "suggest changes" feature in GH!

```suggestion
In ``pkg_resources.normalize_path``, additional path normalization is now performed to ensure path values to a directory is always the same, preventing false positives when checking scripts have a consistent prefix to set up on Windows.
```

Reply:
I also changed the mood and voice from imperative to present passive indicative while I was in there.

I think we haven't been terribly consistent about the tense and voice and such of the news fragments. I think maybe this weekend we should discuss it and I can write up something about the grammar (though generally it's not that important that we ask contributors to do it, because when preparing a release we can edit the `CHANGES.rst` for consistency after all the fragments are assembled).